mdnDocsOverrides.json

```
@@ -255,7 +263,9 @@
   ],
   "baseline-shift": [],
   "baseline-source": [],
-  "battery": [],
+  "battery": [
+    "Web/API/Navigator/getBattery"
+  ],
```

Let's also link to the overall API page.

```suggestion
  "battery": [
    "Web/API/Battery_Status_API",
    "Web/API/Navigator/getBattery"
  ],
```